File: notifications/rest/bindings/create-binding/create-binding.js
// NOTE: This example uses the next generation Twilio helper library - for more
// information on how to download and install this version, visit
// https://www.twilio.com/docs/libraries/node
var accountSid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
var authToken = 'your_auth_token';
var Twilio = require('twilio');

var client = new Twilio(accountSid, authToken);
var service = client.notify.v1.services('ISxxx');

service.bindings.create({
  endpoint: 'xxx',
  identity: 'bob',
  bindingType: 'gcm',
  address: 'xxx',
  tag: ['premium', 'new user']
}).then(function(response) {
  console.log(response);
}).catch(function(error) {
  console.log(error);
});

File: notifications/rest/bindings/create-binding/create-binding.java
import com.twilio.sdk.Twilio;
import com.twilio.sdk.creator.notifications.v1.service.BindingCreator;
import com.twilio.sdk.resource.notifications.v1.service.Binding;

public class CreateBinding {
  // Find your Account Sid and Token at twilio.com/user/account
  public static final String ACCOUNT_SID = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
  public static final String AUTH_TOKEN = "{{ auth_token }}";

  public static void main(String args[]) {
    // Initialize the client
    Twilio.init(ACCOUNT_SID, AUTH_TOKEN);

    // Create a binding
    BindingCreator creator = Binding.create("ISxxx","xxx","Bob", Binding.BindingType.GCM,"xxx");
    creator.setTag("premium");
    Binding binding = creator.execute();
    System.out.println(binding.toString());
  }
}